lib/ndr_import/file/excel.rb

```diff
@@ -66,10 +66,10 @@ def excel_rows(workbook, sheet_name, &block)
       # Iterate through an xls sheet line by line, yielding each one in turn.
       def xls_rows(workbook, sheet_name)
         return enum_for(:xls_rows, workbook, sheet_name) unless block_given?
-
         return unless workbook.first_row(sheet_name)
-        rows    = workbook.first_row(sheet_name)..workbook.last_row(sheet_name)
-        columns = workbook.first_column(sheet_name)..workbook.last_column(sheet_name)
+
+        rows     = workbook.first_row(sheet_name)..workbook.last_row(sheet_name)
+        columns  = workbook.first_column(sheet_name)..workbook.last_column(sheet_name)
```

Comment:
There are no longer any functional changes going on here, it's just whitespace changes? Worth reverting this file?

Reply:
Thnaks, sorted in 2670078